ktor-client/ktor-client-plugins/ktor-client-logging/common/src/io/ktor/client/plugins/logging/Logging.kt

```diff
@@ -31,6 +40,12 @@ public class LoggingConfig {
 
     private var _logger: Logger? = null
 
+    /**
+     * If true, turns on the [OkHttp logging format](https://github.com/square/okhttp/blob/master/okhttp-logging-interceptor/src/main/kotlin/okhttp3/logging/HttpLoggingInterceptor.kt#L56).
+     * Writes only application-level logs because the low-level HTTP communication is hidden within the engine implementations.
+     */
+    public var okhttpFormat: Boolean = false
```

Comment:
I'm still not sure if it's a good idea to have a library-specific flag. Let's have an enum.